xr_sample_project/Assets/SampleViewer/Samples/XRTableTop/readme.md

```diff
@@ -0,0 +1,67 @@
+# Explore the World In Table Top VR
+
+Move around in an ArcGIS World using Virtual Reality to move and fly through different cities
+![Image of XR Table Top](XRTableTop.png)
+
+## How to use the sample
+
+There are two ways to use this headset, Desktop VR and Android VR. This sample utilizes the OpenXR framework and supports the following headsets/devices for **Desktop VR**:
```

Comment:
> And again, should we use VR or XR?

VR, XR include AR, MR, and VR

Reply:
But this sample can run as both VR or MR, right?

Reply:
Yes but it is platform dependent. We should only change it to MR when talking specifically for android deployment.

Reply:
> Instead of Desktop and Android, I think we should use tethered and standalone (or something along those lines). Desktop VR is ambiguous and may refer to a virtual world experience using a regular monitor.

I don't feel strongly about this but just wanted to make sure it didn't go unnoticed in between other comments.